app/features/analysis/AnalysisTab.js

```diff
@@ -11,9 +13,24 @@ class AnalysisTab extends Component {
         return (
             <ScrollView style={{flex: 1}}>
                 <OneRMScreen />
+                <Image style={styles.pseudoScrollView} source={require('app/appearance/images/blank.png')} />
             </ScrollView>
         );
     }
 }
 
+// Hack for svg bug with scrolling in victory charts on Android
+// https://github.com/FormidableLabs/victory-native/issues/96
```

Comment:
- [ ]  move comment to above `class AnalysisTab` as it describes more than just the stylesheet